charts/hono/templates/hono-adapter-amqp/hono-adapter-amqp-vertx-secret.yaml

```diff
@@ -18,6 +18,12 @@ metadata:
   {{- include "hono.metadata" $args | nindent 2 }}
 type: Opaque
 stringData:
+  logging-quarkus-prod.yml: |
+    {{- .Files.Get "config/logging-quarkus-prod.yaml" | nindent 4 }}
+  logging-quarkus-dev.yml: |
+    {{- .Files.Get "config/logging-quarkus-dev.yaml" | nindent 4 }}
+  logging-quarkus-trace.yml: |
+    {{- .Files.Get "config/logging-quarkus-trace.yaml" | nindent 4 }}
```

Comment:
This block could be put inside a
```
{{- if ( contains "quarkus" .Values.honoImagesType ) }}
```
block. Same for the other secret-yamls.

Reply:
That is true, but does it make a real difference? We are talking about a few bytes ...

Reply:
Right, we can keep it.